GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -184,9 +185,9 @@ private void InitializeComponent()
             this.toolbarCommit.SuspendLayout();
             this.commitStatusStrip.SuspendLayout();
             this.SuspendLayout();
-            //
+            // 
```

Comment:
Why was extra spaces added here?
VS may remove that at next update creating noise...
It is often preferred to not apply such changes.